### ISSUE TITLE
Unit-Test fails

### ISSUE THREAD

**Issue by ThomasRoehl** [assistify/Rocket.Chat#111]
The following Unit-tests are still failing with release 0.59.0:
They are located in `tests/end-to-end/ui/`

05-channel-creation(public channel, private channel)

```
1) [Channel creation] public channel: it should create a public channel:
     Uncaught Error: element (".create-channel__content input[name="name"]") still not visible after 10000ms
      at SideNav.createChannel (tests/pageobjects/side-nav.page.js:98:20)
      at Context.<anonymous> (tests/end-to-end/ui/05-channel-creation.js:46:12)
      at node_modules/chimp/dist/lib/utils/fiberize.js:29:22
```

06-messaging ([Public Channel], [Private Channel], [Direct Message])

```
1) [Message] [Public Channel] "before all" hook:
     Uncaught Error: element (".create-channel__content input[name="name"]") still not visible after 10000ms
      at SideNav.createChannel (tests/pageobjects/side-nav.page.js:98:20)
      at Context.<anonymous> (tests/end-to-end/ui/06-messaging.js:280:13)
      at node_modules/chimp/dist/lib/utils/fiberize.js:29:22
```

09-channel ([Channel])

```
1) [Channel] "before all" hook:
     Uncaught Error: element (".create-channel__content input[name="name"]") still not visible after 10000ms
      at SideNav.createChannel (tests/pageobjects/side-nav.page.js:98:20)
      at Context.<anonymous> (tests/end-to-end/ui/09-channel.js:18:12)
      at node_modules/chimp/dist/lib/utils/fiberize.js:29:22

```

**Comment by mrsimpson** [comment]
fixed with b40fcb47af1ed8a600e46e9c5ace4e22acd90a9d

**Comment by mrsimpson** [comment]
`Duration: 37 min 59 sec`
